byzcoin/bcadmin/main.go

```diff
@@ -692,6 +715,49 @@ func darcRuleDel(c *cli.Context, d *darc.Darc, action string, signer *darc.Signe
 	return nil
 }
 
+func qrcode(c *cli.Context) error {
+	bcArg := c.String("bc")
+	if bcArg == "" {
+		return errors.New("--bc flag is required")
+	}
+
+	out := c.String("out")
+	if out == "" {
+		return errors.New("--out flag is required")
```

Comment:
Please make --out not required, and if not provided, write to os.Stdout (you will need to use qrgo.Write, which takes an io.Writer as it's second parameter). This makes it more Unixy, which always ends up having unexpected benefits down the road. (see http://www.tldp.org/LDP/GNU-Linux-Tools-Summary/html/x12429.htm#UNIX-TOOLS-FURTHER-READING)

Or another interesting behavior with n --out would be, like CISC, to print the QR code in the terminal directly...

Reply:
I do not see any other exported methods in qrgo than NewQR and OutputTerminal.

For now, the bcadmin will thus only be able to print the result into terminal, and the user can save the output using the ">" and "cat" commands